shell/platform/darwin/ios/framework/Source/FlutterPlatformViews_Internal.h

```diff
@@ -424,6 +432,10 @@ class FlutterPlatformViewsController {
 
 // Get embedded view
 - (UIView*)embeddedView;
+
+// The FlutterTouchInterceptingView assigns `flutterAccessibilityContainer` as its
```

Comment:
Instance methods don't need to say that they are about the class they are on; this can just say "Sets `flutterAccessibilityContainer` as this view's `accessibilityContainer`."